services/contract_get_info.proto

```diff
@@ -122,7 +122,7 @@ message ContractGetInfoResponse {
         /**
          * The tokens associated to the contract
          */
-        repeated TokenRelationship tokenRelationships = 11;
+        repeated TokenRelationship tokenRelationships = 11 [deprecated = true];
```

Comment:
For each deprecation, should we add to the field comment about why it's deprecated and what is the alternate source of information? Or at least link to the HIP?

Reply:
Added some justification + link to HIP.